src/CostFunction3D.cpp

```diff
@@ -2670,8 +2668,8 @@ void CostFunction3D::calcHmatrix()
 
 void CostFunction3D::Htransform(const real* Cstate, real* Hstate)
 {
-  uint64_t i;           // uint32_t
-  uint64_t j,begin,end; // uint64_t
+  uint64_t i;          
+  uint64_t j,begin,end;
```

Comment:
Is it enough to use `uint32_t` for `j,begin,end` since the JH vector is 32-bit integer type?

Reply:
See above.